src/SFA.DAS.Reservations.Api.Client/Types/ValidationReservationMessage.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SFA.DAS.Reservations.Api.Client.Types
+{
+    public class ValidationReservationMessage
+    {
+        public long AccountId { get; set; }
```

Comment:
AccountId and ReservationId are in the route, so their inclusion in the message seems superfluous